### PR TITLE
Fe dev kx19

### PR DESCRIPTION
**Pull request by wolf1128**

### CODE REVIEW

File: code/client/src/modules/GeneralPages/Map/ExploreMap.tsx [hunk 1]
```diff
@@ -50,6 +51,14 @@ const ExploreMap = (props: any) => {
           url="https://server.arcgisonline.com/ArcGIS/rest/services/Reference/World_Boundaries_and_Places/MapServer/tile/{z}/{y}/{x}"
         />
 
+        {MunicipalityArea.map((area: Area, index: number) => (
```
Comment (matteovpetrera) [inline code/client/src/modules/GeneralPages/Map/ExploreMap.tsx:54]: don't get what this do, it's not visualized on the map

File: code/client/src/modules/UrbanPlanner/AddDocumentForm/elements/Georeference/GeoreferenceTypeSelection.tsx
Comment (matteovpetrera) [inline code/client/src/modules/UrbanPlanner/AddDocumentForm/elements/Georeference/GeoreferenceTypeSelection.tsx:1]: Need to add alerts if the user submits when he press Point or Area and don't select any of the options. (Should receive error, instead it correctly submits)

File: code/client/src/modules/GeneralPages/Map/elements/MunicipalityArea.ts
Comment (matteovpetrera) [inline code/client/src/modules/GeneralPages/Map/elements/MunicipalityArea.ts:1]: this is not correctly visualized and docs assigned to municipality area are still in one single coordinate

File: code/client/src/modules/UrbanPlanner/AddDocumentForm/elements/Georeference/minimap/MiniMapListAreaModal.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  setPolygonCoordinates: (coords: [number, number][]) => void;
+}
+
+const MiniMapListAreaModal = ({
+  setPolygonName,
+  showPolygonMap,
+  setShowPolygonMap,
+  setPolygonCoordinates,
+}: Props) => {
+  const [listPoint, SetList] = useState<Georeference[]>([]);
```
Comment (matteovpetrera) [inline code/client/src/modules/UrbanPlanner/AddDocumentForm/elements/Georeference/minimap/MiniMapListAreaModal.tsx:22]: this should be listAreas

**Review by simonemarcos — APPROVED** [review]

**Pull request merged**